src/manifest/dbtProject.ts

```diff
@@ -1184,11 +1159,82 @@ select * from renamed
         mappedNode[key] = node;
       }
     }
-    const bulkSchemaResponse = await this.getBulkSchema(
-      bulkSchemaRequest,
-      cancellationToken,
+
+    const dbSchemaRequest = bulkSchemaRequest.filter(
```

Comment:
Nit: each operation could be extracted to indivual functions to avoid bloating a single function